### ISSUE TITLE
wasm-bindgen dependency in Cargo.toml is out of date

### ISSUE THREAD

**Issue by reillysiemens** [rustwasm/wasm_game_of_life#33]
**Describe the bug**
Without an update of the wasm-bindgen dependency from v0.2.13 to v0.2.17 the initial `npm run build-debug` on the `chapter-zero` branch fails to compile.

**To Reproduce**
1. `git clone https://github.com/rustwasm/wasm_game_of_life.git`
2. `cd wasm_game_of_life`
3. `git checkout -b chapter-zero origin/chapter-zero`
4. `npm install`
5. `npm run build-debug`

**Expected behavior**
`npm run build-debug` compiles without error.

**Screenshots**
![npm-run-build-debug-error](https://user-images.githubusercontent.com/3084059/44450246-4758f800-a5a5-11e8-9a3e-4575b647f22d.png)

**Additional context**
It's no trouble to run `cargo update -p wasm-bindgen`, but I think that slows readers down. Luckily the error messages are very clear, but it might still be intimidating to a newcomer to troubleshoot this.

For what it's worth, I'm using
- **nvm** - v0.33.11
- **Node.js** - v10.9.0
- **npm** - v6.4.0
- **cargo** - 1.29.0-nightly (6a7672ef5 2018-08-14)
- **wasm-bindgen-cli** - v0.2.17

**Comment by mgattozzi** [comment]
Hey @reillysiemens to avoid this we should put 0.2 as the dep for wasm-bindgen in the Cargo.toml that way it always uses the newest version and not include a Cargo.lock. Can you confirm this works for the code as is today?

**Comment by reillysiemens** [comment]
Yeah, it looks like that'll work just fine, @mgattozzi. I'd be happy to submit that as a fix if you like. :grin:
![removing-cargo-lock-works](https://user-images.githubusercontent.com/3084059/44623199-90e36480-a87d-11e8-8523-00f1bc653ad2.png)

**Comment by JOE1994** [comment]
Since `Cargo.toml` in the master branch specifies the `wasm-bindgen` dependency to be **0.2**,
this issue is resolved in the master branch and can now be closed :+1:
The project still builds and runs successfully as of today (04/29/2020)
